doc/mob_db.txt

```diff
@@ -278,3 +278,4 @@ DamageTakenRate:
 ViewData:
   Overrides the default view data sent to the client with the given values for:
     Sprite, Weapon, Shield, Robe, HeadTop, HeadMid, HeadLow, HairStyle, BodyStyle, HairColor, BodyColor, Gender, Options
+  Note: HairStyleId will only default to 1, if the ViewData block is defined.
```

Comment:
```suggestion
  Note: HairStyleId will only default to 1, if the ViewData block is not defined.
```
perhaps you meant this?
or alternatives
> When not specified, becomes 1.

just like how others field's default values are mentioned in the doc.

Reply:
The sentence is correct. Please read the [request](https://github.com/HerculesWS/Hercules/pull/2622#discussion_r373753837) made by @dastgirp.

Reply:
@Emistry Currently, it's coded like, it will be 0 if ViewData block is not defined, but if defined, default value is 1

Reply:
hmm ... perhaps i understand it in a wrong way then.
these are what I thought it meant if:

**if ViewData block is defined, a random values**
```
{
   HairStyleId: 123
}
```
then the result will be override as `1` regardless of what value i defined there?
because the doc said it default to 1 if defined. Does it meant the value can always be `1` only, and not using the value provided in the doc? 
> Note: HairStyleId will only default to 1, if the ViewData block is defined.

Doc sound confusing to me, perhaps just my english aren't good to read it.
currently its sound like ... if I defined a values there, then the value will always be `1`, regardless what I put there.


**if ViewData block is defined, but 0 value**
```
{
   HairStyleId: 0
}
```
then result will override `0`. 

**if ViewData block is not defined**
```
{
   not defined
}
```
then result will override as `1`  (default).


but aren't document should be mentioning this scenario?
** if ViewData block is not defined, value default to 1**
```
{
   not defined
}
```
then result will override as `1`  (default).

Reply:
or does libconfig actually allow us to define config in such a way?
```
{
   HairStyleId: 
}
```
without a value? (only define a field without value)
I thought it always need a value, just like normal JSON kind of format.

Reply:
```c
{
 ViewData: {
  // Other fields except Hairstyle
 }
}
```
 ^ In this case, Hairstyle 1

```c
{
 // no view Data block
}
```
 ^ In this case, Hairstyle 0


```c
{
 ViewData: {
  // Other fields
  HairStyle: 123
 }
}
```
 ^ In this case, Hairstyle 123

Reply:
oic, thx for the info